expan/core/early_stopping.py

```diff
@@ -38,7 +38,14 @@ def obrien_fleming(information_fraction, alpha=0.05):
 
 def make_group_sequential(spending_function='obrien_fleming', estimated_sample_size=None, alpha=0.05, cap=8):
     """ A closure to the group_sequential function. """
-    return lambda x, y: group_sequential(x, y, spending_function, estimated_sample_size, alpha, cap)
+    def go(x, y, x_denominators=1, y_denominators=1):
+
+        # these next too lines are wrong, but they are bug-compatible with v0.6.13 !
+        x = x / np.nanmean(x_denominators)
+        y = y / np.nanmean(y_denominators)
```

Comment:
@aaron-mcdaid-zalando question for my understanding: why we need that devisions?

Reply:
The current system, the "weighting trick", simply divides each numerator by the mean of the denominators. That is what I am replicating here.

For `fixed_horizon`, I'm doing the right thing. For the other three (including this one) I'm simply replicating the existing behaviour.

The big picture is that the interface to a worker has been changed:
  - for a direct kpi, pass the two samples as-is
  - for derived kpis, pass four samples, i.e. the numerators and denominators for control and also for treatment